R/generate_simulated_data.R

```diff
@@ -328,20 +328,59 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     uot + ot + ht
   )[(uot + 1):(uot + ot + ht)]
```

Comment:
compute this from the `model_hosp_subpop_over_n`, values weighting by population? Should be equivalent (and if it's not there's a problem; indeed, this would make for a good unit test (confirming that convolving and weighted-summing is equivalent to weighted-summing and then convolving)

Reply:
I actually will add the unit test but then won't use this --- the expected total admissions will be the sum of the observed admissions in each subpop